src/main/java/seedu/address/model/meeting/meeting/Meeting.java

```diff
@@ -49,8 +50,8 @@ public Meeting(Title title, OptionalDescription description, From from, To to,
      * Returns true if From is earlier than To chronologically.
      */
     public static boolean isValidFromAndTo (From from, To to) {
```

Comment:
The error message for this prob needs to change also cos "to" is no longer a date.